### DISCUSSION TITLE
simple front end design from 1v to 400v peak to peak

### DISCUSSION THREAD

**Discussion by santlaha** [fhdm-dev/scoppy#6] [Scoppy Front end designs]
**front end simplified.
** input impedance increased to >1Meg 
** accidental over voltage protection upto 600V**
**single point voltage selection
** linearity adjustment 
mode 0 ==>3v peak to peak(-1.5v to +1.5v)
mode 1 ==>12v peak to peak(-6v to +6v)
mode 2 ==>100v peak to peak(-50v to +50v)
mode 3 ==>400v peak to peak  (-200 to +200V)

mode 1
![tinadiag](https://user-images.githubusercontent.com/86060827/122684408-db9aac00-d222-11eb-82b1-eeb8bc98281e.jpg)
[sim_12vmax.zip](https://github.com/fhdm-dev/scoppy/files/6682868/sim_12vmax.zip)

mode 3
[tinadiag 400v.zip](https://github.com/fhdm-dev/scoppy/files/6682907/tinadiag.400v.zip)
![tinadiag 400v](https://user-images.githubusercontent.com/86060827/122685004-95dfe280-d226-11eb-87f3-e0633b230bbd.jpg)

**Comment by gilesp1729** [comment]
Hi,
Could you publish a web-readable schematic?
